src/stores/Binding/Store.ts

```diff
@@ -219,6 +224,76 @@ const initializeAndGenerateUUID = (
     };
 };
 
+const hydrateConnectorTagDependentState = async (
+    connectorTagId: string,
+    get: StoreApi<BindingState>['getState']
+): Promise<ConnectorTagResourceData | null> => {
+    if (!hasLength(connectorTagId)) {
+        return null;
+    }
+
+    const { data, error } = await getSchema_Resource(connectorTagId);
+
+    if (error) {
+        get().setHydrationErrorsExist(true);
+    } else if (data?.resource_spec_schema) {
+        const { setBackfillSupported, setResourceSchema } = get();
```

Comment:
Just access directly.

Reply:
This is an oversight given the PR was opened before it was reviewed. This PR is older than that which changed how the state getter is interacted with.